flow-model-generator/modules/flow-model-generator-ls-extension/src/main/java/io/ballerina/flowmodelgenerator/extension/ProjectCacheManager.java

```diff
@@ -36,27 +38,27 @@
 public class ProjectCacheManager {
 
     private final Path sourceDir;
-    private final Path filePath;
     private Path destinationProjectPath;
-    private Path destinationPath;
 
-    public ProjectCacheManager(Path sourceDir, Path filePath) {
+    public ProjectCacheManager(Path sourceDir) {
```

Comment:
Shall we make the constructor visibility protected. Otherwise there is no guarantee of there is only one instance per path.